src/main/java/io/rebloom/client/Client.java

```diff
@@ -196,8 +196,16 @@ public boolean exists(String name, byte[] value) {
   /**
    * Remove the filter
    * @param name
+   * @return true if delete the filter, false is not delete the filter
    */
-  public void delete(String name) {
-    _conn().getClient().del(name);
+  public boolean delete(String name) {
+      try(Jedis conn = _conn()){
+          return conn.del(name) != 0;
+      }
+  }
+
+  public void close(){
```

Comment:
shouldn't Client implement Closeable?

Reply:
public abstract class Pool<T> implements Closeable;

this is use Pool's close method, It's implement Closeable already.

Reply:
@gkorland  are you have any question?

Reply:
I think the Client should also implement Closeable

Reply:
yeah, @gkorland, after my consideration, I agree with you. 
because of implement Closeable can implicit call close method in try code block.

Reply:
@gkorland please merge pr, thanks!